sdk/cognitiveservices/cognitiveservices-luis-runtime/README.md

```diff
@@ -17,124 +17,143 @@ npm install @azure/cognitiveservices-luis-runtime
-    options: {
-      datetimeReference: new Date().toISOString(),
-      overridePredictions: true
-    },
-    externalEntities: [{
+import { CognitiveServicesCredentials } from "@azure/ms-rest-azure-js";
+import { LUISRuntimeClient } from "@azure/cognitiveservices-luis-runtime";
+
+let authoringKey = process.env["luis-authoring-key"];
+const creds = new CognitiveServicesCredentials(authoringKey);
+
+// check the following link to find your region
+// https://docs.microsoft.com/en-us/azure/cognitive-services/luis/luis-reference-regions
+const region = "<your-region>";
+const client = new LUISRuntimeClient(creds, "https://" + region + ".api.cognitive.microsoft.com/");
+
+const appId = "<your-app-id>"; // replace this with your appId.
+const versionId = "0.1"; // replace with version of your luis application. Initial value will be 0.1
+
+const predictionRequest = {
+  query: "testquery",
+  options: {
+    datetimeReference: new Date(),
+    overridePredictions: true
+  },
+  externalEntities: [
+    {
       entityName: "testentityName",
       startIndex: 1,
       entityLength: 1,
       resolution: {}
-    }],
-    dynamicLists: [{
+    }
+  ],
+  dynamicLists: [
+    {
       listEntityName: "testlistEntityName",
-      requestLists: [{
-        name: "testname",
-        canonicalForm: "testcanonicalForm",
-        synonyms: ["testsynonyms"]
-      }]
-    }]
-  };
-  const verbose = true;
-  const showAllIntents = true;
-  const log = true;
-  client.prediction.getVersionPrediction(appId, versionId, predictionRequest, verbose, showAllIntents, log).then((result) => {
+      requestLists: [
+        {
+          name: "testname",
+          canonicalForm: "testcanonicalForm",
+          synonyms: ["testsynonyms"]
+        }
+      ]
+    }
+  ]
+};
+const verbose = true;
+const showAllIntents = true;
+
+client.prediction
+  .getVersionPrediction(appId, versionId, predictionRequest, { verbose, showAllIntents })
+  .then((result) => {
     console.log("The result is:");
     console.log(result);
+  })
+  .catch((err) => {
+    console.error(err);
   });
-}).catch((err) => {
-  console.error(err);
-});
 ```
 
 #### browser - Authentication, client creation and getVersionPrediction prediction as an example written in JavaScript.
 
-##### Install @azure/ms-rest-browserauth
+##### Install @azure/ms-rest-azure-js
 
 ```bash
-npm install @azure/ms-rest-browserauth
+npm install @azure/ms-rest-azure-js
 ```
 
 ##### Sample code
 
-See https://github.com/Azure/ms-rest-browserauth to learn how to authenticate to Azure in the browser.
-
 - index.html
+
 ```html
 <!DOCTYPE html>
 <html lang="en">
   <head>
     <title>@azure/cognitiveservices-luis-runtime sample</title>
-    <script src="node_modules/@azure/ms-rest-js/dist/msRest.browser.js"></script>
-    <script src="node_modules/@azure/ms-rest-browserauth/dist/msAuth.js"></script>
+    <script src="node_modules/@azure/ms-rest-azure-js/dist/cognitiveServicesCredentials.js"></script>
```

Comment:
```suggestion
    <script src="node_modules/@azure/ms-rest-js/dist/msRest.browser.js"></script>
```

The `@azure/ms-rest-azure-js` package does not have a browser bundle and I have logged https://github.com/Azure/ms-rest-azure-js/issues/126 to track this issue.

Until that issue is fixed, we can use the `ApiKeyCredentials` from `@azure/ms-rest-js` instead.